packages/create-pantheon-decoupled-kit/src/templates/next-wp-search-api-addon/lib/Search.js

```diff
@@ -0,0 +1,52 @@
+import { gql } from '@pantheon-systems/wordpress-kit';
+import { client } from './WordPressClient';
+
+export async function getSearchResults(searchTerm) {
```

Comment:
Some context on what is happening with this query:

When accessing the Solr index via graphql, we are confined to the fields exposed in our graphql schema. Since our does not offer a way to hit both posts and pages in the same place, I have two separate statements in the query below. One for posts and one for pages, I then concatenate the results together for the return value.

While there are some extra steps here, the response object keeps it's correct scored and ordered shape. Solr scores pages below posts. Because of this, any post that contains a match to the search term should (and does) show before the highest ranked page.

This query (not using graphql) shows how posts will come before pages in scoring when looking at the Solr index, what I have below is replicating the same behavior. https://dev-wp-search-demo.pantheonsite.io//wp-json/wp/v2/search?search=example

Since we are bringing pages into the mix now I decided to break this out into it's own `Search.js` file as I don't think `Posts.js` suits it anymore.

Reply:
Thanks for the explainer, seems good to me!